bmi_sdk/tests/test_bmi_time.py

```diff
@@ -0,0 +1,29 @@
+from ..bmi_time import BmiTime
+from pydantic import ValidationError
+import pytest
+
+@pytest.mark.parametrize('unit', ["s", "sec", "second", "seconds", 
```

Comment:
The interchange between single and double quotes is going to make my hair grey faster hahah, but I figured you might do that on purpose lol. On a more serious note, what are your thoughts on using something like `black` in CI to enforce a certain formatting style? I could get it set up with pre-commit easily so you really don't have to think about it. Just nice to have things auto-formatted for you even if you don't always like exactly how it formats things.

Reply:
I'm 💯 ok with that haha.  Formatting can hard to get correct when the focus in on semantics.

Reply:
Sweet, just opened up #24 to add this. Ill add it to CI once this has merged.